R/article.R

```diff
@@ -249,6 +249,15 @@ peerj_article <- function(..., keep_tex = TRUE) {
   pdf_document_format("peerj",  keep_tex = keep_tex, ...)
 }
 
+#' @section \code{pihph_article}: Format for creating submissions to the
+#'   Papers in Historical Phonology. Adapted from
+#'   \url{https://github.com/pihph/templates}.
```

Comment:
I would also add a link to the journal's website directly as you put in the README. This is helpful for users

Reply:
eb006ee